spec/models/services/facebook_spec.rb

```diff
@@ -45,7 +77,7 @@
 
       @photos.each(&:save!)
 
-      @status_message = alice.build_post(:status_message, :text => "the best pebble.")
+      @status_message = alice.build_post(:status_message, text: "the best pebble.")
```

Comment:
You can move everything from this line and below to the `it "should include post url in message with photos"`, because it is only used once there. That way this code doesn't run a second time for `it "should include post url in message with photos when no text message"` where it isn't needed (because you create a message without text there).

Reply:
ok